client/container/Market.jsx

```diff
@@ -0,0 +1,148 @@
+
+import Actions from '../core/Actions';
+import Component from '../core/Component';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import HorizontalRule from '../component/HorizontalRule';
+import Table from '../component/Table';
+import BitShares from 'btsdex-fix';
+import CircularProgress from '@material-ui/core/CircularProgress';
+class Market extends Component {
+  static defaultProps = {
+    coin: {}
+  };
+
+static propTypes = {
+    coin: PropTypes.object.isRequired  
+};
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      cols: [
+        { key: 'price', title: 'BTC' },
+        { key: 'quantity', title: 'BWK' },      ],
+      orderbookasks:[],orderbookbids:[],orderdataret:[],loading : true,newrefresh: false
+    };
+  };
+
+  fetchOrderBook = async(update) => {
+    if(update){this.setState({newrefresh : true})}
+    try {
+      let quote = "BWK"
```

Comment:
Let's move these to config

Reply:
sure, ill update it to use the config vars

Reply:
Added code to use the coindetails coinname instead of hardcoding it in,should be ok